packages/dso-toolkit/src/components/document-card/document-card.models.ts

```diff
@@ -0,0 +1,17 @@
+import { Label } from "../label/label.models.js";
+import { Toggletip } from "../toggletip/toggletip.models.js";
+
+export interface DocumentCard {
+  label: string;
+  href: string;
+  active?: boolean;
+  meta?: Label;
+  type: string;
+  typeToelichting: Toggletip<never>;
+  status: string;
+  dsoDocumentCardClick?: (e: CustomEvent<DsoDocumentCardClickEvent>) => void;
+}
+
+export interface DsoDocumentCardClickEvent {
+  originalEvent?: MouseEvent;
```

Comment:
Ik denk dat deze ook onterecht optioneel is gemarkeerd

Reply:
Optionaliteit is weg